modules/openxoutstreamBidAdapter.js

```diff
@@ -80,8 +83,9 @@ function buildCommonQueryParamsFromBids(bid, bidderRequest) {
     dddid: bid.transactionId,
     openrtb: '%7B%22mimes%22%3A%5B%22video%2Fmp4%22%5D%7D',
     nocache: new Date().getTime(),
-    vht: bid.params.height || bid.sizes[0][1],
-    vwd: bid.params.width || bid.sizes[0][0]
+    vht: height,
```

Comment:
ok, added a note to the MD file!

Reply:
> > @HolzAndrew - so let me see if I understand -- if a user specifies an AdUnit of size 300x250 and includes openxoutstream as an adapter, it will under the covers change the size to 400x300 and make the auction request to your backend.
> > The net result being that a request for 300x250 could end up returning a creative of a totally different size? This bid will get thrown out due to size mismatch.
> > If I'm understanding this correctly, you need to fix this behavior.
> > If 400x300 is the only size you support, that's fine, but you should avoid making auctions unless the AdUnit specifies that size. Should not be weighing down the browser with requests/responses that are going to get thrown out due to a size mismatch.
> 
> @HolzAndrew @bretg
> 
> My understanding is that the sizes are a reflection of OpenX’s demand, and will display within the size of the placement created by the publisher.
> 
> So in the situation you described — the openx-outstream adapter would make a request to OpenX for video demand up to that size (400x300). The Yieldmo format then dynamically resizes the video within the creative to fit within the ad unit size set up by the publisher (300x250 in this case).

@harquail Exactly, thats the gist. (@bretg )